### DISCUSSION TITLE
New service for time tracking and accounting

### DISCUSSION THREAD

**Discussion by Paul-B98** [helpwave/helpwave#20] [Ideas]
The focus of this request is to create a service for recording the employee’s time. This is import because of the court decision [1 ABR 22/21 of the BAG from the 13.09.2022](https://www.bundesarbeitsgericht.de/presse/einfuehrung-elektronischer-zeiterfassung-initiativrecht-des-betriebsrats/) which enforces a service to track the time of employee’s. 

Additionally, there should be a possibility to keep track of the Vacation. It should also be possible to schedule shifts for the employees according to the regulation needs. Days of because of illnesses should be tracked too. 

Another extension for this idea is to add an accounting feature for the payment of the Salaries of the employees.

**Comment by Paul-B98** [comment]
@aegis301 fyi

**Comment by aegis301** [comment]
This is a very intersting topic to dive into and of high current legal relevance. However, for now I think it's best for us to focus our scope on individual health personell, as well as groups like a team or even an entire department. A time tracking service for a whole hospital might be out of scope _for now_?

Another concern, for me is the fact that (and this is highly subjective), for now, I see helpwave as a flexible startup building flexible, fast and helpful software, marketed to the individual and usable without the whole weight of the hospital infrastructure on it's shoulders. Diving into accounting and salaries carries with it the need to be integrated into the hospital ecosystem which can be cumbersome. It could even yield some negative feedback from healthcare workers to be billed through helpwave software as I will elaborate below.

The other point would be shift scheduling, vacation planning and tracking. This is very intersting and would yield some amazing integration potential with helpwave tasks! I find this very intersting and we should definately follow this path in the future. To pick up your previous point concerning salaries and accounting: In my experience you as a doctor constantly have to track your own hours, shifts etc. to check if your billing statements at the end of the month add up. This is a time consuming task and in the jungle of different accounting laws, tarif contracts and different shifts it can lead to health care workers loosing actual money

We could use this tool to facilitate tracking hours and what doctors and nurses *should* be paid to give them back control over their finances. This could be marketed again to the individual as the rest of our software suite.
